build/Packaging.cake

```diff
@@ -2,17 +2,25 @@
 
 var version = Argument("version", "0.4.0");
 
+Task("Build-ObjC")
+    .Does(() =>
+    {
+        StartProcess("make", new ProcessSettings {
+        Arguments = new ProcessArgumentBuilder().Append ("-C").Append ("objcgen/").Append ("nuget-prep")
+	});
+    });
+
 Task("Create-Package")
     .IsDependentOn("Build-Binder")
+    .IsDependentOn("Download-Xamarin-Android")
```

Comment:
I think these targets are right, Build-Binder and Download-Xamarin-Android are what we need.